source/reference/mongosync/mongosync-behavior.txt

```diff
@@ -140,16 +140,15 @@ Consistency
 ~~~~~~~~~~~
 
 ``mongosync`` supports :term:`eventual consistency` on the destination
-cluster. Read Consistency is not guaranteed on the destination cluster,
-the source and destination clusters may differ at a given point in
-time. However, if new writes are halted on the source cluster, the
-source and destination clusters will eventually hold the same data.
+cluster. Read consistency is not guaranteed on the destination cluster until 
+commit. Before committing, the source and destination clusters may differ at a 
+given point in time. To learn more, see :ref:`mongosync-considerations`.
 
-Individual writes may be reordered or combined as they are relayed from
-source to destination. For a given document, the total number of writes
-may differ between source and destination.
+While ``mongosync`` is syncing, individual writes may be reordered or combined 
```

Comment:
can we rephrase this sentence to be "mongosync may reorder or combine writes as it relays them from source to destination."